src/backend/distributed/executor/multi_utility.c

```diff
@@ -1650,8 +1650,10 @@ ErrorIfUnsupportedAlterTableStmt(AlterTableStmt *alterTableStatement)
 			}
 
 			case AT_DropConstraint:
+			case AT_EnableTrigAll:
+			case AT_DisableTrigAll:
```

Comment:
Are there any other commands that we can trivially add here?

e.g. 
- `AT_EnableTrigUser`
- `AT_DisableTrigUser`
- `AT_SetLogged`
- `AT_SetUnlogged`
- `AT_ChangeOwner`